great-idea-website/index.html

```diff
@@ -13,51 +13,88 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
-
+<header>
+    <nav>
+        <a href= "#">Services </a>
+        <a href= "#">Product </a>
+        <a href="#">Vision</a>
+        <a href="#" >Features</a>
+        <a href="#">About</a>
+        <a href= "#">Contact</a>
+    </nav>
+
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+
+  </header>
   <body>
+    
+       
+
+</div>
+
+<div>
+
+  <header>
```

Comment:
you only want to have one header and this would be it. the Nav doesnt need to be the header. It is just the nav and can have its own div